src/replica/Controller.cc

```diff
@@ -101,15 +101,15 @@ void tracker(weak_ptr<Controller> const& controller, string const& context) {
                     if (prevWorkerInfo != workerInfo) {
                         LOGS(_log, LOG_LVL_INFO,
                              context << "worker '" << workerInfo.name << "' logged in from host '"
-                                     << workerInfo.svcHost
+                                     << workerInfo.svcHost.addr
```

Comment:
Why not log both name and address?

Reply:
Good point. This could be easily fixed by reverting the change. The overloaded `operato<<` will print them both.